bg_atlasgen/metadata_utils.py

```diff
@@ -129,10 +131,12 @@ def create_metadata_files(dest_dir, metadata_dict, structures, root_id):
     :param uncompr_atlas_path: path to uncompressed atlas folder
     :param metadata_dict: dict with atlas metadata
     :param structures: list of dictionaries with structures hierarchical info
+    :param additional_metadata: Dict to add to atlas metadata
     """
     # write metadata dict:
     with open(dest_dir / descriptors.METADATA_FILENAME, "w") as f:
-        json.dump(metadata_dict, f)
+        # only save additional metadata to json, don't include in readme
+        json.dump(metadata_dict | additional_metadata, f)
```

Comment:
is this the bit that is python 3.9 only? Is it really worth asking developers to upgrade atlasgen env to 3.9?